src/main/resources/schemas/nhinc/common/configAdmin.xsd

```diff
@@ -28,73 +28,81 @@
     </xsd:complexType>
     <xsd:element name="ImportCertificateRequest" type="tns:ImportCertificateRequestType" />
         
-    <xsd:complexType name="ImportCertificateResponseMessageType">
+    <xsd:complexType name="SimpleResponseMessageType">
         <xsd:sequence>
             <xsd:element name="status" type="xsd:boolean"/>
             <xsd:element name="message" type="xsd:string"/>
         </xsd:sequence>
     </xsd:complexType>
-    <xsd:element name="ImportCertificateResponseMessage" type="tns:ImportCertificateResponseMessageType"/>
+    <xsd:element name="ImportCertificateResponseMessage" type="tns:SimpleResponseMessageType"/>
     
+	<xsd:complexType name="ListKeyStoresRequestMessageType" >
+        <xsd:sequence>
+            <xsd:element name="configAssertion" type="tns:ConfigAssertionType" minOccurs="1" maxOccurs="1" />
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="ListKeyStoresRequestMessage" type="tns:ListKeyStoresRequestMessageType" />
+    
+    <xsd:complexType name="ListKeyStoresResponseMessageType">
+         <xsd:sequence>
+            <xsd:element name="certList" type="tns:ListCertificateType" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="ListKeyStoresResponseMessage" type="tns:ListKeyStoresResponseMessageType"/>
     
     <xsd:complexType name="ListTrustStoresRequestMessageType" >
         <xsd:sequence>
             <xsd:element name="configAssertion" type="tns:ConfigAssertionType" minOccurs="1" maxOccurs="1" />
+			<xsd:element name="refreshCertCache" minOccurs="1" maxOccurs="1" default="false" />
         </xsd:sequence>
     </xsd:complexType>
     <xsd:element name="ListTrustStoresRequestMessage" type="tns:ListTrustStoresRequestMessageType" />
     
     <xsd:complexType name="ListTrustStoresResponseMessageType">
          <xsd:sequence>
-            <xsd:element name="certData" type="xsd:base64Binary" xmime:expectedContentTypes="application/octet-stream" minOccurs="0" maxOccurs="unbounded"/>
+            <xsd:element name="certList" type="tns:ListCertificateType" minOccurs="0" maxOccurs="unbounded" />
```

Comment:
I think you should only have 1 list.  The unbounded should be the cert elements in the list.

Reply:
I think this is just a naming issue.  I would keep it as certData not CertList

Reply:
There's a variable called certData in ListCertificateType element. Hence named as certList.. 
The unbounded data here is a combination of alias and certdata. we can't have unbounded just for certdata?

Reply:
Alias information is part of KeyStore/TrustStore and it is not part of Certificate. So, the response has alias and cert data combined. This combined information forms the information for one certificate and hence having unbounded for this tuple.